docs/testing/ie-11-testing.md

```diff
@@ -1,44 +1,50 @@
+>
+> For more information about these options, see [Support Internet Explorer 11](../develop/support-ie-11.md).
+>
+> Also, Internet Explorer 11 does not support some HTML5 features such as media, recording, and location.
+
+> [!NOTE]
+> To test how your add-in runs in Office on the web in Internet Explorer, simply open Office on the web in Internet Explorer and [sideload the add-in](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md).
+
+## Prerequisites
 
 - [Node.js](https://nodejs.org/) (the latest [LTS](https://nodejs.org/about/releases) version)
-- A code editor. We recommend [Visual Studio Code](https://code.visualstudio.com/)
-- [Be part of the Office Insider program](https://insider.office.com)
 
 These instructions assume you have set up a Yo Office generator project before. If you haven't done this before, consider reading a quick start, such as [this one for Excel add-ins](../quickstarts/excel-quickstart-jquery.md).
 
-## Using IE11 tooling
+## Switching to the Internet Explorer 11 webview
 
 1. Create a Yo Office generator project. It doesn't matter what kind of project you select, this tooling will work with all project types.
 
-> ![NOTE]
-> If you have an existing project and want to add this tooling without creating a new project, skip this step and move to the next step. 
-
-2. In the root folder of your new project, run the following in the command line:
+    > [!NOTE]
+    > If you have an existing project and want to add this tooling without creating a new project, skip this step and move to the next step. 
 
-```command&nbsp;line
-npx office-addin-dev-settings webview manifest.xml ie
-```
-You should see a note in the command line that the web view type is now set to IE.
+1. In the root folder of your project, run the following in the command line. (This example assumes that your project's manifest file is in the root. If it isn't, specify the relative path to the manifest file.) You should see a message in the command line that the web view type is now set to IE.
 
-> ![TIP]
-> It isn't necessary to use this tooling, but it should help debug the majority of issues related to the Internet Explorer 11 runtime. For complete robustness, you should test using a computer with a copy of Windows 7 and Office 2013 installed.
+    ```command&nbsp;line
+    npx office-addin-dev-settings webview manifest.xml ie
+    ```
 
-## Command settings
+> [!TIP]
+> It isn't necessary to use this command, but it should help debug the majority of issues related to the Internet Explorer 11 runtime. For complete robustness, you should test using computers with various combinations of Windows 7, 8, and 10 and various versions of Office. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

Comment:
Should we link to that article about getting older versions of Office from here?